robot/lib/Src/drivers/ICM20948.cpp

```diff
@@ -0,0 +1,189 @@
+#include "drivers/ICM20948.hpp"
```

Comment:
I'm conflicted. It's really standard when doing these drivers to just have the functionality for everything exposed and let the user do all the config. But having a second wrapper class or placing the config into the module really dirties up the repo. This is probably the best way around that, but it feels wrong from an embedded pov

Reply:
I haven't checked the actual driver yet, I'll do it another time